releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,7 +1,16 @@
-## Microsoft.Extensions.Configuration.AzureAppConfiguration
-### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
+# Microsoft.Extensions.Configuration.AzureAppConfiguration
+[Source code][source_code] | [Package (NuGet)][package]
 
-### 7.0.0 - November 21, 2023
+## 7.1.0-preview - January 19, 2024
+### Enhancements
+* The provider will now automatically discover other existing replicas and attempt to connect to them when the AppConfig store and any replicas passed by the user have failed. This is enabled by default and can be disabled by setting the new `AzureAppConfigurationOptions.ReplicaDiscoveryEnabled` property equal to `false`. [#442](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/442)
```

Comment:
```suggestion
* The provider will now automatically discover replicas and use them in fail-over scenarios. This is enabled by default and can be disabled by setting the new `AzureAppConfigurationOptions.ReplicaDiscoveryEnabled` property equal to `false`. [#442](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/442)
```

I don't think the ordering bit is too important.

Reply:
This is so important I might even suggest it to be it's own section.
 
Like

### Replica auto discovery

The provider now supports automatic discovery of replicas. ...

Reply:
Yea I think that's good point, I could create a new section and add a more detailed summary like what @zhenlan suggested if you agree with that.

Reply:
Yes, sounds good.